src/Passport.php

```diff
@@ -715,4 +717,17 @@ public static function withoutCookieEncryption()
 
         return new static;
     }
+
+    /**
+     * Specify which view should be used as the authorization view.
+     *
+     * @param  callable|string  $view
+     * @return void
+     */
+    public static function authorizationView($view)
+    {
+        app()->singleton(AuthorizationViewResponseContract::class, function ($app) use ($view) {
```

Comment:
Passport doesn't require laravel/framework, so the `app()` helper might not be present

```suggestion
        \Illuminate\Container\Container::getInstance()->singleton(AuthorizationViewResponseContract::class, function ($app) use ($view) {
```

Reply:
Every first party package is only compatible with the framework so this is fine.